datumaro/plugins/icdar_format/extractor.py

```diff
@@ -59,16 +60,13 @@ def _load_recognition_items(self):
                 objects = line.split(', ')
                 if len(objects) == 2:
                     image = objects[0]
-                    objects = objects[1].split('\"')
-                    if 1 < len(objects):
-                        if len(objects) % 2:
-                            captions = [objects[2 * i + 1]
-                                for i in range(int(len(objects) / 2))]
+                    captions = []
+                    for caption in objects[1:]:
+                        if caption[0] != '\"' or caption[-1] != '\"':
+                            log.warning('Captions must be inside double quotes, \
```

Comment:
The `else` branch contradicts this message. Maybe, it should tell that a pairing quote is not found?